archive/c/c-plus-plus/rot13.cpp

```diff
@@ -0,0 +1,28 @@
+#include <iostream>
+#include <string>
+
+using namespace std;
+
+void rot13(string& str) {
+    for (char& c : str) {
+        if (('a' <= c && c <= 'z') || ('A' <= c && c <= 'Z')) {
+            if (('a' <= c && c <= 'm') || ('A' <= c && c <= 'M')) {
+                c += 13;
+            } else {
+                c -= 13;
+            }
+        }
+    }
+}
+
+int main(int argc, char* argv[]) {
+    if (argc == 2 && string(argv[1]).size() != 0 && !isdigit(argv[1][0])) {
```

Comment:
Why is the first character of the string to encrypt checked to see if it is not a digit?
```suggestion
    if (argc == 2 && string(argv[1]).size() != 0) {
```
Digits should be allowed.

Reply:
How would be the output in case of digits?